tasks.py

```diff
@@ -850,7 +853,7 @@ def unittest_coverage(context):
         "lint-only": "Only run linters; unit tests will be excluded. (default: False)",
     }
 )
-def tests(context, failfast=False, keepdb=False, lint_only=False):
+def tests(context, failfast=False, keepdb=False, lint_only=False, buffer=True, pdb=False):
```

Comment:
We need to update the help dict in the decorator for this function and the `unittest` function to add the new arguments